python/dgl/distributed/partition.py

```diff
@@ -0,0 +1,163 @@
+"""Functions for partitions."""
```

Comment:
1. i would prefer to expose the file names of a partition to users, instead of hard code the names in the code. it's more flexible. the json file can also tell users more information what is going on with the partitioning.
2. edge id mapping in stored EID. when you say node id mapping, do you mean edge assignment?
3. added checks for each field.